x-pack/winlogbeat/module/sysmon/config/winlogbeat-sysmon.js

```diff
@@ -361,7 +372,21 @@ var sysmon = (function () {
         evt.Delete("winlog.event_data.DestinationIsIpv6");
     };
 
-    var addHashes = function (evt, hashField) {
+    var setRelatedIP = function (evt) {
+        var sourceIP = evt.Get("source.ip");
+        if (sourceIP && net.isIP(sourceIP)) {
```

Comment:
`isIP` is unnecessary since you are copying from `source.ip` a value that has already been validating by the `convert` processor and `type: ip`.